client/db_test.go

```diff
@@ -288,3 +290,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]bool{
+		"AdminMerge":          true,
+		"AdminSplit":          true,
+		"Commit":              true,
+		"DebugName":           true,
+		"InternalAddCall":     true,
+		"InternalKV":          true,
+		"InternalSetPriority": true,
+		"Run":                  true,
+		"SetDebugName":         true,
+		"SetSnapshotIsolation": true,
+		"Tx": true,
+	}
+	for _, typ := range types {
+		for j := 0; j < typ.NumMethod(); j++ {
+			m := typ.Method(j)
+			var ch rune
```

Comment:
`ch, _ := utf8.DecodeRune(m.Name)`

Or, according to http://golang.org/pkg/reflect/#Method, `m.PkgPath == ""` iff `m` is exported.